docs/aws-copilot.md

```diff
@@ -0,0 +1,116 @@
+# Provisioning a new environment using AWS Copilot
+
+## Configure AWS local credentials
+
+Install `awscli` and `aws-copilot`. Then configure your AWS CLI credentials
+locally:
+
+```bash
+$ aws configure sso
+```
+
+Use the SSO Start URL and SSO Region from the "Command line or programmatic
+access" link in the AWS Account admin.
+
+Once you're done, your `~/.aws/config` should look something like:
+
+```bash
+[default]
+region = eu-west-2
+[profile Admin-ACCOUNT_ID]
+sso_session = fwk
+sso_account_id = ACCOUNT_ID
+sso_role_name = Admin
+region = eu-west-2
+[sso-session fwk]
+sso_start_url = https://SUBDOMAIN.awsapps.com/start#
+sso_region = eu-west-2
+sso_registration_scopes = sso:account:access
+```
+
+You can make the new profile the default by changing the config to:
+
+```diff
+[default]
+-region = eu-west-2
```

Comment:
Maybe remove this line since it's only re-added further below? Or do you reckon position may be significant?